fdbserver/fdbserver.actor.cpp

```diff
@@ -2375,6 +2392,8 @@ int main(int argc, char* argv[]) {
 					pFiles->push_back(f);
 				}
 			}
+			// Update blob proxy string
+			g_network->setGlobal(INetwork::enProxy, opts.proxy);
```

Comment:
This is an interesting way of setting proxy, which seems better than https://github.com/apple/foundationdb/pull/10885 where a global variable is used.

Reply:
I suppose we should use the existing global, g_network->setGlobal(INetwork::enProxy, opts.proxy);, in fdbbackup.actor.h rather than make a new one